retroarch.c

```diff
@@ -8561,13 +8581,23 @@ bool command_event(enum event_command cmd, void *data)
 
             if (grab_mouse_state)
             {
-               if ((ret = input_driver_grab_mouse()))
+               if ((ret = input_st->current_driver &&
+input_st->current_driver->grab_mouse))
+               {
+                  input_st->current_driver->grab_mouse(
+                        input_st->current_data, true);
                   input_st->grab_mouse_state = true;
+               }
             }
             else
             {
-               if ((ret = input_driver_ungrab_mouse()))
+               if ((ret = input_st && input_st->current_driver &&
```

Comment:
This is broken. `ret` was previously being used to check whether `input_driver_grab_mouse()`. What you have copy/pasted here is fundamentally different (and will not work)